optaplanner-core/src/main/java/org/optaplanner/core/impl/partitionedsearch/scope/PartitionChangeMove.java

```diff
@@ -34,12 +34,13 @@
 import org.optaplanner.core.impl.score.director.ScoreDirector;
 
 /**
+ * Applies a new best solution from a partition child solver into the global working solution of the parent solver.
  * @param <Solution_> the solution type, the class with the {@link PlanningSolution} annotation
  */
 public final class PartitionChangeMove<Solution_> extends AbstractMove<Solution_> {
 
     public static <Solution_> PartitionChangeMove<Solution_> createMove(InnerScoreDirector<Solution_> scoreDirector,
-                                                                        int partIndex) {
+            int partIndex) {
```

Comment:
@ge0ffrey Would you mind importing the latest version of [KIE Java Conventions](https://github.com/kiegroup/droolsjbpm-build-bootstrap/tree/7.6.0.Final/ide-configuration/intellij-configuration/code-style) to avoid formatting inconsistency?

Reply:
This needs to happen on all sources of the optaplanner repository in one big bulk change - currently all sources follow the older version of the KIE conventions that I have locally.

The new conventions might have some readability trouble on long method signatures (especially with generic methods), but you're right - it's better to be consistent than to be ideal. Anyway, let's make a separate jira and investigate there.